src/Ledger/Conway/Specification/Utxow.lagda.md

```diff
@@ -157,34 +148,54 @@ data _⊢_⇀⦇_,UTXOW⦈_ where
     ∙  Γ ⊢ s ⇀⦇ tx ,UTXO⦈ s'
        ────────────────────────────────
        Γ ⊢ s ⇀⦇ tx ,UTXOW⦈ s'
-\end{code}
-\end{AgdaMultiCode}
-\caption{UTXOW inference rules}
-\label{fig:rules:utxow}
-\end{figure*}
-\begin{code}[hide]
+```
+
+<!--
+```agda
 pattern UTXOW-inductive⋯ p₁ p₂ p₃ p₄ p₅ p₆ p₇ p₈ h
       = UTXOW-inductive (p₁ , p₂ , p₃ , p₄ , p₅ , p₆ , p₇ , p₈ , h)
 pattern UTXOW⇒UTXO x = UTXOW-inductive⋯ _ _ _ _ _ _ _ _ x
 
 unquoteDecl UTXOW-inductive-premises =
   genPremises UTXOW-inductive-premises (quote UTXOW-inductive)
-\end{code}
-
-\subsection{Plutus script context}
-\hrefCIP{0069}
-unifies the arguments given to all types of Plutus scripts currently available:
-spending, certifying, rewarding, minting, voting, proposing.
-
-The formal specification permits running spending scripts in the absence datums
-in the Conway era.  However, since the interface with Plutus is kept abstract
-in this specification, changes to the representation of the script context which
-are part of \hrefCIP{0069} are not included here.  To provide a \hrefCIP{0069}-conformant
-implementation of Plutus to this specification, an additional step processing
-the \List{} \Data{} argument we provide would be required.
-
-In \cref{fig:rules:utxow}, the line
-\inputHashes{}~\subseteqfield{}~\txdatsHashes{} compares two inhabitants of
-\PowerSet{}~\DataHash{}.  In the Alonzo spec, these two terms would
-have inhabited \PowerSet{}~(\Maybe{}~\DataHash{}), where a \nothing{} is thrown
-out~\parencite[\sectionname~3.1]{alonzo-ledger-spec}.
+```
+-->
+
+
+## Plutus script context
+
+[CIP-0069](https://cips.cardano.org/cip-0069) unifies the arguments
+given to all types of Plutus scripts currently available: spending,
+certifying, rewarding, minting, voting, proposing.
+
+The formal specification permits running spending scripts in the absence
+datums in the Conway era. However, since the interface with Plutus is
+kept abstract in this specification, changes to the representation of
+the script context which are part of
+[CIP-0069](https://cips.cardano.org/cip-0069) are not included here. To
+provide a [CIP-0069](https://cips.cardano.org/cip-0069)-conformant
+implementation of Plutus to this specification, an additional step
+processing the `List`{.AgdaDatatype} `Data`{.AgdaFunction} argument we
+provide would be required.
```

Comment:
It feels like if this paragraph should go in an admonition. At the least it should be swapped with the paragraph that follows for readability.